packages/cli/src/utils/config.ts

```diff
@@ -1,29 +1,72 @@
-import { existsSync, readFileSync, rmdirSync, writeFileSync } from "fs"
+import { existsSync, readFileSync, rmdirSync, writeFileSync, unlinkSync } from "fs"
 import { safeLoad, safeDump } from "js-yaml"
 import { join } from "path"
-import Account from "@mesg/api/lib/account-lcd"
-import merge from "lodash.merge"
+import { toWords, encode } from 'bech32'
+import Account, { bech32Prefix } from "@mesg/api/lib/account-lcd"
 
-const FILE = 'config.yml'
+export type Config = {
+  engine: {
+    authorized_pubkeys: string[],
+    account: {
+      mnemonic: string
+    }
+  }
+  mnemonic: string
+}
+
+const ENGINE_FILE = 'config.yml'
+const CLI_FILE = '.mesgrc'
 
 export const clear = (path: string): void => {
-  (rmdirSync as any)(path, { recursive: true })
+  const rm = rmdirSync as any
+  rm(join(path, 'cosmos'), { recursive: true })
+  rm(join(path, 'tendermint'), { recursive: true })
+  unlinkSync(join(path, ENGINE_FILE))
+  unlinkSync(join(path, CLI_FILE))
 }
 
-export const read = (path: string): any => {
-  return existsSync(join(path, FILE))
-    ? safeLoad(readFileSync(join(path, FILE)).toString())
+const read = (filepath: string): Config => {
+  return existsSync(filepath)
+    ? safeLoad(readFileSync(filepath).toString())
     : {}
 }
 
-export const write = (path: string, config: any) => {
-  const newConfigs = merge({}, read(path), config)
-  writeFileSync(join(path, FILE), safeDump(newConfigs))
+const write = (path: string, config: Config): Config => {
+  writeFileSync(join(path, CLI_FILE), safeDump(config))
+  writeFileSync(join(path, ENGINE_FILE), safeDump(config.engine))
+  return config
+}
+
+const hasTestAccount = (path: string): boolean => {
+  if (!existsSync(join(path, ENGINE_FILE))) return false
+  if (!existsSync(join(path, CLI_FILE))) return false
+  const config = read(join(path, CLI_FILE))
+  return !!config.mnemonic && !!config.engine.account.mnemonic
+}
+
+// https://github.com/forbole/big-dipper/blob/master/imports/startup/server/util.js
+const pubKey = (mnemonic: string): string => {
+  const publicKey = Account.deriveMnemonic(mnemonic).publicKey
+  const pubkeyAminoPrefix = Buffer.from('EB5AE98721', 'hex')
```

Comment:
More information [here](https://github.com/tendermint/go-amino#computing-the-prefix-and-disambiguation-bytes) and [here](https://github.com/forbole/big-dipper/blob/master/imports/startup/server/util.js#L31)